docs/docs/test-runner/overview.md

```diff
@@ -17,7 +17,7 @@ Test runner for web applications.
 ⏱&nbsp;&nbsp;Runs tests in parallel and isolation.<br>
 👀&nbsp;&nbsp; Interactive watch mode.<br>
 🏃&nbsp;&nbsp; Fast development by rerunning only changed tests.<br>
-🚀&nbsp;&nbsp; Powered by [esbuild](/docs/dev-server/plugins/esbuild.md) and [rollup plugins](/docs/dev-server/plugins/rollup.md)
+🚀&nbsp;&nbsp; Powered by [esbuild](/docs/dev-server/plugins/esbuild/) and [rollup plugins](/docs/dev-server/plugins/rollup/)
```

Comment:
Hmm but it breaks now for the GitHub view, would it work for both the website and GitHub if we removed the trailing slash? I'm sort of hoping GitHub will be smart enough to append the .md extension suffix.. If not, we need to either decide that links don't need to work for GitHub, or make a smart prepublish step or something I guess?

Reply:
I don't think that it's currently working for the github view, e.g. `esbuild` links to 

https://github.com/modernweb-dev/web/blob/master/docs/dev-server/plugins/esbuild.md

the correct link however would be

https://github.com/modernweb-dev/web/blob/master/docs/docs/dev-server/plugins/esbuild.md

Reply:
links should always be relative and to md files (then it will be correctly resolved on github and by the static site generator) 👍 

so this should work
```suggestion
🚀&nbsp;&nbsp; Powered by [esbuild](../dev-server/plugins/esbuild.md) and [rollup plugins](../dev-server/plugins/rollup.md)
```